angular/main.js

```diff
@@ -0,0 +1,2 @@
+ng.platformBrowserDynamic.platformBrowserDynamic()
+    .bootstrapModule(AppModule);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['AppModule' is not defined.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5238370)